docs/user/debugging/python.md

```diff
@@ -17,15 +17,15 @@ If the `GLEAN_LOG_PINGS` environment variable is set to `true`, pings are
 logged to the console on `DEBUG` level whenever they are submitted.
 
 Make sure that when you configure logging in your application, you set the
-level for the `glean` logger to `DEBUG` or higher. Otherwise pings won't be
+level for the Python logger to `DEBUG` or higher. Otherwise pings won't be
```

Comment:
Hm, the previous sentence was not completely wrong. As I understand it Glean uses a logger by the name of `glean.rust_core` which can be configured independently from what the application does for its own loggers.

Reply:
The messages coming from the Rust core use `glean.rust_core`, but other messages coming from Python come from loggers named after the Python module they come from (as is Python convention),e.g. `glean.metrics.string` etc.  The crappy bit about Python logging comes into play, though, because there is no way to say "for all loggers with a given prefix, set this level", so there's no way to set the level of all Glean logging at once.  So these old instructions never really worked (except for logging from the top-level module).

We could change the behavior so that everything uses a single logger called `glean` -- it's not really the Python convention, but it feels like the better fit for this library.